takwimu/takwimu_ui/src/Theme.js

```diff
@@ -47,8 +59,7 @@ const Theme = createMuiTheme({
     },
     subtitle1: {
       fontFamily: FONT_FAMILY_TEXT,
-      fontSize: '1.75rem',
-      fontWeight: 'bold',
+      fontSize: '1.25rem',
```

Comment:
This will mess up `Where to Next` section. Looking at the design, this component shouldn't be using `subtitle1`, `body1` is enough (but **bold**),